src/apis/users.api.ts

```diff
@@ -0,0 +1,5 @@
+import { UserPayload, UserResponse } from "../types/User";
+import http from "../utils/http";
+
+export const postUser = (user: UserPayload) =>
+  http.post<UserResponse>(`http://localhost:3000/users/`, user);
```

Comment:
don't hardcode for the url, it should be get from env

Reply:
dạ, em sẽ sửa lại ạ

Reply:
done